src/dbnode/generated/proto/annotation/annotation.proto

```diff
@@ -3,11 +3,17 @@ syntax = "proto3";
 package annotation;
 
 message Payload {
-    MetricType metric_type   = 1;
+    SourceFormat source_format = 3;
+
+    // Used when source_format == PROMETHEUS
+    PrometheusType prom_type = 1;
```

Comment:
nit: As discussed perhaps let's call this `open_metrics_metric_family`?

Reply:
Naming it slightly differently than suggested (`open_metrics_family_type`).